src/main/java/com/hm/achievement/listener/PlaceholderAchievementListener.java

```diff
@@ -0,0 +1,56 @@
+package com.hm.achievement.listener;
+
+import org.bukkit.entity.Player;
+
+import com.hm.achievement.AdvancedAchievements;
+import com.hm.achievement.api.AdvancedAchievementsAPI;
+import com.hm.achievement.category.MultipleAchievements;
+import com.hm.achievement.category.NormalAchievements;
+
+import me.clip.placeholderapi.external.EZPlaceholderHook;
+
+public class PlaceholderAchievementListener extends EZPlaceholderHook
```

Comment:
Is this class actually a listener? I haven't thought this through, but I would be tempted to say it should be called something like `AchievementPlaceholderHook` and moved to the `utils` package.